test/common/common.go

```diff
@@ -148,3 +149,17 @@ func OcManaged(args ...string) (string, error) {
 	}
 	return string(output), err
 }
+
+// update clusterName in checking YAML based clusterNamespace in testing env
+func UpdateClusterName(
```

Comment:
This function is very specific to your test cases as yamlPlc needs to follow certain structure. I suggest to move it to your own test case.